apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md

```diff
@@ -0,0 +1,35 @@
+---
+heading: Vue and HTML examples on Documentation site
+author: Julio Ortega
+posted: '2024-8-13'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+# Hi Dialers 💜
+
+We have a pretty short announcement, but packed with great news! 🥳
+
+## Dialtone documentation
+
+We are currently on an effort to unify our two documentation sites [Dialtone vue - Storybook](https://dialtone.dialpad.com/vue) and [Dialtone](https://dialtone.dialpad.com) into [Dialtone](https://dialtone.dialpad.com) only.
+
+Today, We are happy to announce the availability of **Vue component examples** on the documentation site. That way is easier for you to understand the usage of it, slots, props, events, see some examples and copy the code directly so you can start using our components right away!
```

Comment:
`That way is easier for you to understand the usage` -> `This makes it easier for you to understand the usage`